src/components/Activities/ActivitiesInstance/ActivitiesInstance.jsx

```diff
@@ -99,37 +103,37 @@ export default class extends React.Component {
     }
 
     render() {
+        if (!this.props.startDate) {
+            return null;
+        }
         const date = this.formatDate(
-            this.props.start_date,
-            this.props.end_date,
-            this.props.frequency
-        );
-        const time = this.formatTime(
-            this.props.start_time,
-            this.props.end_time
-        );
-        const chip_time_stamp = this.formatTimeStamp(
-            this.props.start_date,
-            this.props.start_time
+            this.props.startDate,
+            this.props.endDate,
+            [true, false, true, true, true]
```

Comment:
Hva skjer her?

Reply:
Det har med frequency å gjøre (arrangementet på mandag/tirsdag/onsdag osv...) men tror ikke det skal bli med i alphaen ettersom det ikke er laget for når man lager aktiviteten også er det i en annen form i databasen som "recurrencypattern" som er en string.

Reply:
Hmm, ja sliter med å  forstå hva du sier, det jeg lurer på er lom det er mocket siden du manuelt putter inn en false midt i, og hvorfor det er begrenset til ukedagene og ikke tar med helg forøvrig.

Reply:
Det kan vi bare fjerne og heller bare vise dato/tidspunkt for aktiviteten siden det uansett kun er enkeltinstanser. 

Her er forøvrig spesifikasjonen for hvordan vi kommer til å gjøre det med aktiviteter: [https://tools.ietf.org/html/rfc5545](https://tools.ietf.org/html/rfc5545)

Reply:
Interessant :O Da tenker du vel etter demoen? Den så veldig lang og komplisert ut